client/src/components/MyProfile.js

```diff
@@ -221,17 +234,121 @@ function Home({ currentUser, setCurrentUser }) {
     }
   };
 
+  const fetchUsersEvents = async () =>
```

Comment:
I know this isn't used yet but I guess we could already include all the async handling:
![image](https://github.com/jannejjj/RASP24/assets/61980297/77e5ee7a-745f-406f-b00c-f58ad5698d2c)